### PR TITLE
Fix python float issues

### PR DESCRIPTION
**Pull request by Caribosaurus**

### CODE REVIEW

**Comment by Caribosaurus** [comment]
@thecsw

File: src/comment_worker.py [hunk 1]
```diff
@@ -265,20 +265,19 @@ def invest(self, sess, comment, investor, amount, suffix):
...
-                amount = amount * multiplier
-            except ValueError:
-                return
+        multiplier = CommentWorker.multipliers.get(suffix, 1)
+        try:
+            input_amount = int(amount.replace(',', ''))
+            if multiplier == '%':
+                multiplier = float(input_amount/100) if amount < 100 else 1
+                input_amount = int(investor.balance)
+            amount = input_amount * multiplier
```
Comment (thecsw) [inline src/comment_worker.py:278]: ~Do you think we would need to set the multiplier value here to 1 if it doesn't equal to `%`?~

Missed a part. Good stuff

File: src/comment_worker.py [hunk 2]
```diff
@@ -299,9 +298,7 @@ def invest(self, sess, comment, investor, amount, suffix):
         ))
 
         # 0 upvotes is too OP, so what we do is make around 5 minumum
-        upvotes_now = int(comment.submission.ups)
-        if upvotes_now < 5:
-            upvotes_now = 5
+        upvotes_now = 5 if int(comment.submission.ups) < 5 else int(comment.submission.ups)
```
Comment (thecsw) [inline src/comment_worker.py:301]: Good quick refactoring. Thanks!

**Review by thecsw — APPROVED** [review]
I left a comment about uninitialized multiplier

**Pull request merged**